interface/main/holidays/import_holidays.php

```diff
@@ -0,0 +1,197 @@
+<?php
+/**
+ * interface/main/holidays/import_holidays.php holidays/clinic handle import/download holidays files
+ *
+ * Copyright (C) 2016 Sharon Cohen <sharonco@matrix.co.il>
+ *
+ * LICENSE: This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 3
+ * of the License, or (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://opensource.org/licenses/gpl-license.php>;.
+ *
+ * @package OpenEMR
+ * @author  sharonco <sharonco@matrix.co.il>
+ * @link    http://www.open-emr.org
+ */
+
+?>
+<?php
+/**
+ *
+ * @package OpenEMR
+ * @author  sharon cohen <sharonco@matrix.co.il>
+ * @link    http://www.open-emr.org
+ */
+
+set_time_limit(0);
+
+// Disable magic quotes and fake register globals.
+$sanitize_all_escapes = true;
+$fake_register_globals = false;
+require_once('../../globals.php');
+require_once($GLOBALS['srcdir'] . '/acl.inc');
+require_once("$srcdir/htmlspecialchars.inc.php");
+require_once("$srcdir/sql.inc");
```

Comment:
Note the htmlspecialchars.inc.php and sql.inc are already brought in with the above globals.php call